CONTRIBUTING.md

```diff
@@ -60,19 +59,18 @@ Platform-specific guides to setting up a development environment:
 - [How to clone and setup the freeCodeCamp website on a Windows pc](https://forum.freecodecamp.org/t/how-to-clone-and-setup-the-free-code-camp-website-on-a-windows-pc/19366)
 - [How to Clone and Setup the freeCodeCamp Website on a Mac](https://forum.freecodecamp.org/t/how-to-clone-and-setup-the-freecodecamp-website-on-a-mac/78450)
 
-### Forking The Project
+### Forking the Project
 
 #### Setting Up Your System
 
 1. Install [Git](https://git-scm.com/) or your favorite Git client.
 2. (Optional) [Setup an SSH Key](https://help.github.com/articles/generating-an-ssh-key/) for GitHub.
-3. Create a parent projects directory on your system. For this guide, it will be assumed that it is `/mean/`
 
 #### Forking freeCodeCamp
 
 1. Go to the top level freeCodeCamp repository: <https://github.com/freeCodeCamp/freeCodeCamp>
 2. Click the "Fork" Button in the upper right hand corner of the interface ([More Details Here](https://help.github.com/articles/fork-a-repo/))
-3. After the repository has been forked, you will be taken to your copy of the FCC repo at `yourUsername/freeCodeCamp`
+3. After the repository (repo) has been forked, you will be taken to your copy of the freeCodeCamp repo at `yourUsername/freeCodeCamp`
```

Comment:
nit: lets use `https://github.com/yourUsername/freeCodeCamp.git`

Reply:
😕 Do you mean https://github.com/yourUsername/freeCodeCamp without the .git instead of `yourUsername/freeCodeCamp`?

Reply:
Oh yes. without the `.git`

Reply:
That's a good idea, less confusing. ✔️